### ISSUE TITLE
Inventory for I-Rvat MS Capp. Sist. 57

### ISSUE THREAD

**Issue by juliacmcf** [diamm/diamm#263]
I have no idea how to fix this:
https://www.diamm.ac.uk/admin/diamm_site/problemreport/83408/change/

**Comment by ahankinson** [comment]
I'm getting a page-not-found. Did you delete the problem report (there were a lot of duplicates; I suspect they're double-clicking the 'submit' button.

**Comment by juliacmcf** [comment]
Yes, he duplicated lots; the last two from him refer to the same issue…

**Comment by ahankinson** [comment]
It looks like the source just needs an inventory? The entry he is referring to is from the CCM, so we probably shouldn't change that. 

In lieu of a 'full' inventory, couldn't you just copy his note and put it as a 'Contents' note on the source, saying that it's an update from the CCM entry?

**Comment by juliacmcf** [comment]
I have added his note as a DIAMM note and credited him. However there is a larger issue (literally) as if we want to add an inventory to this MS (or indeed any large MS) adding 143 pages manually is going to be a nightmare. I can't see a way around this, and it's partly addressed by being able to add pages at the bottom of the source description.

**Comment by ahankinson** [comment]
No longer a problem.
